src/Microsoft.Performance.SDK/Processing/IProcessingSource.cs

```diff
@@ -25,14 +25,14 @@ public enum SerializationSource
     public interface IProcessingSource
     {
         /// <summary>
-        ///     Gets the collection of tables exposed by this <see cref="IProcessingSource"/>.
-        ///     These are the data tables for exposing the processed data.
+        ///     Gets the collection of tables exposed by this <see cref="IProcessingSource"/>
+	///     which are not marked as <see cref="TableDescriptor.IsMetadataTable"/>
```

Comment:
>	///  [](http://example.com/codeflow?start=0&length=5)

nit: in CodeFlow, it looks like indentation is off for this, and below

Reply:
yeah it is. edited outside of visual studio :/